javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -1933,7 +1933,7 @@ public static LiteralTree getDefaultValueTree(
       case BYTE:
         return TreeUtils.createLiteral(TypeTag.BYTE, (byte) 0, typeMirror, processingEnv);
       case CHAR:
-        return TreeUtils.createLiteral(TypeTag.CHAR, '\u0000', typeMirror, processingEnv);
+        return TreeUtils.createLiteral(TypeTag.CHAR, (int) '\u0000', typeMirror, processingEnv);
```

Comment:
The literal values for booleans and chars should be stored as integers because they are converted from integers to booleans or chars  before they are returned by `LiteralTree#getValue`.  (The changes in this file are a separate bug that should probably be in a separate pull request.)

Reply:
This is a separate pull request (https://github.com/typetools/checker-framework/pull/5391).